src/jsonapi-vuex.js

```diff
@@ -261,6 +260,8 @@ const actions = (api) => {
       let action = api
         .delete(path, config)
         .then((result) => {
```

Comment:
Typo: `result` not `results`. (Though given `get`/`post`/`patch` use `results` I'm happy for all references here to change for consistency.)